app/controllers/corporate_information_pages_controller.rb

```diff
@@ -47,9 +39,7 @@ def set_slimmer_headers_for_document
 
   def find_organisation
     @organisation =
-      if params.key?(:organisation_id)
-        Organisation.friendly.find(params[:organisation_id])
-      elsif params.key?(:worldwide_organisation_id)
+      if params.key?(:worldwide_organisation_id)
```

Comment:
Could the `if` be removed here as I'm assuming `WorldwideOrganisation.friendly.find(params[:worldwide_organisation_id])` would return an error if there's no `worldwide_organisation_id` anyway?

Reply:
Good point, thanks. Have removed!